media-autobuild_suite.bat

```diff
@@ -1930,3 +1973,7 @@ if not exist %instdir%\msys64\mingw%1\bin\gcc.exe (
 )
 endlocal
 goto :EOF
+
+:resolvePath
+set "resolvePath=%~dpnx1"
+goto :EOF
```

Comment:
please preserve the newline at EOF

Reply:
Damn I wondered whether there is supposed to be one, it just didn't look like there was any on GitHub:

![](https://user-images.githubusercontent.com/4985349/198180606-71b15568-cf06-4ce8-9b5a-89109dd8ac0f.png)

Reply:
Done